paasta_tools/cli/cmds/status.py

```diff
@@ -805,17 +864,94 @@ def _print_flink_status_from_job_manager(
         else:
             output.append(f"    Flink version: {flink_config.flink_version}")
 
+        # Print Flink Dashboard URL
         # Annotation "flink.yelp.com/dashboard_url" is populated by flink-operator
         dashboard_url = metadata["annotations"].get("flink.yelp.com/dashboard_url")
         output.append(f"    URL: {dashboard_url}/")
 
+    if verbose:
+        # Print Flink config link resources
+
+        try:
+            # Convert the yelp_region to ecosystem
+            # https://yelpwiki.yelpcorp.com/spaces/PRODENG/pages/14226200/Habitat+Datacenter+Ecosystem+Runtimeenv+Region+Superregion
+            # Example
+            #  convert_location_type(
+            #      location="uswest2-devc",
+            #      source_type="region",
+            #      desired_type="ecosystem",
+            #  )
+            # Output: devc
+            kube_clusters_data = system_paasta_config.get_kube_clusters()
+            cluster_info = kube_clusters_data.get(cluster)
+            yelp_region = cluster_info.get("yelp_region", None)  # uswest2-devc
+
+            ecosystem = convert_location_type(
+                location=yelp_region,
+                source_type="region",
+                desired_type="ecosystem",
+            )[0]
+        except Exception as e:
+            ecosystem = "pnw-prod"  # Default ecosystem
```

Comment:
should we just let this crash? if any of the bits above fail - something is almost certainly wrong with paasta's configuration

Reply:
Updated in 68b42b6c9cd3a111190e92340b6895db9faf276e